### DISCUSSION TITLE
4 lane project

### DISCUSSION THREAD

**Discussion by Nissesnik** [micbay/RaceTimerAndController#26] [Show and tell]
Hi and thanks very much for sharing this projekt.
Excellently documented and explained.

We are four Swedish men in their 60s, one of whom had an old car track lying around (Märklin Sprint).
Since everyone is competitive, some form of timing device must be developed.
I found this project that was perfect for our four-lane track.
Now we can have a beer and run our races without having to argue about the results.

![IMG_1743](https://github.com/user-attachments/assets/6b7c2341-ac4b-42e9-8a5d-d846a932e9a5)
![IMG_1747](https://github.com/user-attachments/assets/00318b2f-5aac-495f-b344-0dbc5dcd929a)
![IMG_1748](https://github.com/user-attachments/assets/b8111fe6-3897-45cd-bbd0-80e3d1ded7b5)
![IMG_1751](https://github.com/user-attachments/assets/1ad23fe5-4ed7-4b3e-9fff-c58418817885)

https://github.com/user-attachments/assets/4e9743a8-dc14-48af-89a5-40f70184f20d

https://github.com/user-attachments/assets/67bcbb11-d09d-4d31-ac20-05522a574f6b

**Comment by micbay** [comment]
Skål gentlemen! Nice setup, our first posted 4-laner. Many happy races to you all, thanks for sharing,
